pandas/core/indexes/base.py

```diff
@@ -7475,7 +7475,9 @@ def ensure_index_from_sequences(sequences, names=None) -> Index:
     """
     from pandas.core.indexes.multi import MultiIndex
 
-    if len(sequences) == 1:
+    if len(sequences) == 0:
+        return Index([])
```

Comment:
Could you use `default_index(0)` here? Can import it from `pandas.core.indexes.api`

Reply:
@mroeschke,

That's not working since `pandas.core.indexes.api` imports `pandas.core.indexes.base` on [L19](https://github.com/pandas-dev/pandas/blob/main/pandas/core/indexes/api.py#L19) so I get the following error:

```
Traceback (most recent call last):
  .
  .
  .
  File "/home/op/git/pandas/pandas/core/indexes/base.py", line 177, in <module>
    from pandas.core.indexes.api import default_index
ImportError: cannot import name 'default_index' from partially initialized module 'pandas.core.indexes.api' (most likely due to a circular import) (/home/op/git/pandas/pandas/core/indexes/api.py)
```

Just out of curiosity why is, `defaut_index(0)` prferred over `Index([])`?

Reply:
You'll probably need to import it in the method.

`default_index` is, as the name implies, the "default" implementation of an index used in constructors if not explicitly specified

Reply:
Thanks for clarifying.